dts/bindings/clock/ite,it8xxx2-ecpm.yaml

```diff
@@ -0,0 +1,49 @@
+# Copyright (c) 2022 ITE Corporation. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+description: ITE it8xxx2 ECPM (EC Clock and Power Management Controller) node
+
+compatible: "ite,it8xxx2-ecpm"
+
+include: [clock-controller.yaml, base.yaml]
+
+properties:
+    reg:
+      required: true
+
+    label:
+      required: true
+
+    pll-frequency:
+      type: int
+      required: true
+      enum:
+            - 0
+            - 1
+            - 2
+      description: |
+        0 = PLL_48_MHZ,
+        1 = PLL_96_MHZ,
+        2 = PLL_DEFAULT,
+
+    mcu-clock:
+      type: int
+      required: true
+      enum:
+            - 96000000
+            - 48000000
+            - 24000000
+            - 12000000
+      description: |
+        96000000 = CLK_CPU_96MHZ,
+        48000000 = CLK_CPU_48MHZ,
+        24000000 = CLK_CPU_24MHZ,
+        12000000 = CLK_CPU_12MHZ,
+
+    "#clock-cells":
+      const: 3
+
+clock-cells:
+  - offset
+  - bitmask
+  - always
```

Comment:
I suggest naming "always_on_bitmask" for clarity.

Reply:
Got it.